backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/services/PictureService.java

```diff
@@ -0,0 +1,34 @@
+package com.nokia.library.nokiainnovativeproject.services;
+
+import com.cloudinary.Cloudinary;
+import com.cloudinary.Singleton;
+import com.cloudinary.utils.ObjectUtils;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class PictureService {
+    public Map uploadPicture(MultipartFile picture) throws IOException {
+
+        validateFile(picture);
+        Map uploadResult = null;
+        if (picture != null) {
+            Cloudinary cloudinary = Singleton.getCloudinary();
```

Comment:
Where is configuration for this cloundary service?

Reply:
It's in a environment variable. Should I hardcode it for development purposes?